<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1061</width>
     <height>691</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>주문목록</string>
    </attribute>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>631</width>
       <height>591</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>매출</string>
    </attribute>
    <widget class="QTableView" name="tableView_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>651</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendar">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>20</y>
       <width>272</width>
       <height>183</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_btn">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>초기화</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>220</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_btn">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>설정</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSum">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
